ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -43,15 +43,26 @@
 
 struct azure_file_system;
 struct azure_file_handle;
+
+// Statistics to be collected for the Azure blob storage.
+struct azure_statistics {
+    uint64_t list_objects_count;  // Number of list objects requests
+    uint64_t put_object_count;    // Number of put object requests
+    uint64_t read_object_count;   // Number of get object requests
+    uint64_t object_exists_count; // Number of object exists requests
```

Comment:
I think the comments here (and in the GCP equivalent) are a bit redundant - the naming is clear enough already.

Reply:
done! also renamed the variables [fcb039f](https://github.com/wiredtiger/wiredtiger/pull/8932/commits/fcb039f1e2c4c0dbf1e68bb8dad0aff3bebc9f24)